packages/react-form/src/start/utils.ts

```diff
@@ -1,3 +1,15 @@
-import { createCookie } from '@remix-run/node'
+import { getCookie, setCookie } from '@tanstack/react-start/server'
+import * as devalue from 'devalue'
 
-export const _tanstackInternalsCookie = createCookie('_tanstack_form_internals')
+export const _tanstackInternalsCookie = {
```

Comment:
Methods here should be `get` and `set`, right?

Reply:
idk coz getter and setter has performance drawback when we use in V8. getter/setter is really ergonomic but it is internal function so I avoided it

Reply:
I'm also fine with parse/save but right now you defined parse/save in this file but in the other files you're using get/set which obviously makes the compiler unhappy :)

Reply:
yeah off topic! Thanks